README.md

```diff
@@ -19,3 +19,17 @@ the following features:
     - Pieces of text can be addressed specifically by underlining this text
     - Comments in float enviroments (e.g., figure)
 
+  - Fonts
+    - The [ACMart fonts](https://github.com/opencompl/paper-template/blob/master/acmart.cls#L720-L728)
+      use [Inconsolata](https://ctan.org/pkg/inconsolata?lang=en) for monospace, 
+      [Libertine](https://ctan.org/pkg/libertine?lang=en) for serif, and [newtx](https://ctan.org/pkg/newtx?lang=en) for math.
+    - The Libertine `ttf` can be obtained as "linux libertine" on distros, and Inconsolata `ttf` as "Inconsolata".
+    - The default font size of the body is `11pt`.
+  - Figures
+    - The paper is of US letter size, which in *physical units* is [width:`8.5in`, length:`11in`](https://en.wikipedia.org/wiki/Letter_(paper_size))
+    - Choose figure size in *physical units* (`mm`/`inch`) based on how much of US letter size it occupies and export to PDF.
+    - Import figure PDF into paper using `\includegraphics{path/to/figure}`.
+      This occupies the desired space in the paper; there is no need to use
+      `\includegraphics[width=<insert-width-here>]{path/to/figure}`. 
+    -  `matplotlib` [by default uses physical units when sizing a figure](https://matplotlib.org/stable/gallery/subplots_axes_and_figures/figure_size_units.html#figure-size-in-inches-default).
+       Thus, when creating a figure, use `fig = plt.figure(figsize=(3, 6))` for a `3inx6in` figure. 
```

Comment:
I am wondering if it makes sense to add this information (in parts) into `plots/plot.py`? In particular, can we give some typical figure sizes, e.g. \textwidth on the acmsmall layout?

Reply:
@tobiasgrosser how about providing this information within the template as part of the appendix?

In LaTeX, you can use `\the\columnwidth` to print the column width in points, e.g., for the ACM `sigplan` document class parameter (double column) this is 240.94499pt, then one can use this directly in GDraw to set the canvas size (it supports points) and also in matplotlib for a plot size that fits one single column.

We only need a handful of such quantities from [here](https://en.wikibooks.org/wiki/LaTeX/Page_Layout#Page_dimensions).